src/styles/blocks/star.css

```diff
@@ -0,0 +1,33 @@
+.rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
```

Comment:
no margin on blocks

Reply:
Same, it it's element styles - move them to corresponding block.